ci/lava/qcs6490-rb3gen2-core-kit.yml

```diff
@@ -0,0 +1,47 @@
+actions:
+- deploy:
+    images:
+      image:
+        headers:
+          Authentication: Q_GITHUB_TOKEN
+        url: "{{BUILD_DOWNLOAD_URL}}"
+    postprocess:
+      docker:
+        image: ghcr.io/foundriesio/lava-lmp-sign:main
+        steps:
+        - export IMAGE_PATH=$PWD
+        - echo "DEVICE_TYPE=qcs6490-rb3gen2" >> $IMAGE_PATH/flash.settings
+        - cat $IMAGE_PATH/flash.settings
+    timeout:
+      minutes: 10
+    to: downloads
+- deploy:
+    images:
+      image:
+        url: downloads://core-image-minimal-{{DEVICE_TYPE}}.rootfs.qcomflash.tar.gz
```

Comment:
FWIW. we have a PR to switch to core-base-image , which should be merged soon.

Reply:
this is a limitation in LAVA. I can't pass a variable here. Maybe setting a file name in workflow is a better idea.

Reply:
fixed